src/components/splash/Splash.js

```diff
@@ -7,32 +7,36 @@ import Section from '../common/layout/Section'
 import Config from '../../config/config'
 import { getDesignRelativeHeight } from '../../lib/utils/sizes'
 
+const isDemo = Config.isPhaseZero === 'true'
+
 const Splash = ({ animation }) => (
   <Wrapper style={styles.wrapper}>
     <Section style={styles.container}>
       <View style={styles.waves} />
       <Section.Stack style={styles.content} grow justifyContent="center">
-        <Section.Stack>
-          <Section.Text
-            fontSize={26}
-            fontWeight="bold"
-            color="white"
-            letterSpacing={0.13}
-            lineHeight={32}
-            style={styles.title}
-          >
-            GoodDollar Demo
-          </Section.Text>
-          <Section.Text fontSize={16} color="white" letterSpacing={0.24} lineHeight={22} fontWeight="medium">
-            {'All G$ coins in the demo\nare for test purposes only.\nOnce all feedback is incorporated,\n'}
-            <Section.Text fontSize={16} color="white" letterSpacing={0.24} lineHeight={22} fontWeight="bold">
-              all demo G$ coins will be deleted.
+        {isDemo && (
+          <Section.Stack>
+            <Section.Text
+              fontSize={26}
+              fontWeight="bold"
+              color="white"
+              letterSpacing={0.13}
+              lineHeight={32}
+              style={styles.title}
+            >
+              GoodDollar Demo
```

Comment:
```suggestion
              GoodDollar {isPhaseZero && 'Demo'}
```

Reply:
"gooddollar" title should be kept according to the XD

Reply:
Gooddollar title it a part of animation